src/platform/qt/GameController.cpp

```diff
@@ -622,8 +622,9 @@ void GameController::closeGame() {
 	if (!m_gameOpen) {
 		return;
 	}
-
+#ifdef USE_GDB_STUB
 	setDebugger(nullptr);
```

Comment:
If `setDebugger` isn't present when `USE_GDB_STUB` isn't defined, that's the real problem; the CLI debugger should still function without the GDB stub.